recipes/mTEDx/ASR/Transducer/hparams/train_pruned.yaml

```diff
@@ -0,0 +1,226 @@
+# ############################################################################
+# Model: E2E ASR with attention-based ASR
+# Encoder: CRDNN model
+# Decoder: GRU + beamsearch + RNNLM
+# Tokens: BPE with unigram
+# losses: Transducer
+# Training: mTEDx-fr
+# Authors:  Mohamed Anwar 2022
+# ############################################################################
+
+# Seed needs to be set at top of yaml, before objects with parameters are made
+seed: 1234
+__set_seed: !!python/object/apply:torch.manual_seed [!ref <seed>]
+output_folder: !ref ./results/mTEDx_fr/CASCADE/CRDNN_pruned
+wer_file: !ref <output_folder>/wer.txt
+save_folder: !ref <output_folder>/save
+
+# Language model (LM) pretraining
+# NB: To avoid mismatch, the speech recognizer must be trained with the same
+# tokenizer used for LM training. Here, we download everything from the
+# speechbrain HuggingFace repository. However, a local path pointing to a
+# directory containing the lm.ckpt and tokenizer.ckpt may also be specified
+# instead. E.g if you want to use your own LM / tokenizer.
+pretrained_tokenizer_path: #!PLACEHOLDER
+pretrained_lm_path: #!PLACEHOLDER
+
+# Data files
+data_folder: #!PLACEHOLDER
```

Comment:
Can be `!PLACEHOLDER` directlly (w/o '#').

Reply:
fixed, thanks for the heads-up!